ICSharpCode.Decompiler/TypeSystem/IParameter.cs

```diff
@@ -105,4 +105,34 @@ public interface IParameter : IVariable
 		/// </summary>
 		IParameterizedMember? Owner { get; }
 	}
+
+	internal static class IParameterExtensions
```

Comment:
We already have `TypeSystemExtensions`, wouldn't it make sense to keep them all in the same unit?

Reply:
I can move it over